src/connectors/configure/__tests__/connectConfigure-test.js

```diff
@@ -13,18 +13,22 @@ describe('connectConfigure', () => {
     helper = algoliasearchHelper(fakeClient, '', {});
   });
 
-  describe('throws on bad usage', () => {
-    it('without searchParameters', () => {
+  describe('Usage', () => {
+    it('without searchParameters throws', () => {
       const makeWidget = connectConfigure();
       expect(() => makeWidget()).toThrow();
     });
 
-    it('with a renderFn but no unmountFn', () => {
-      expect(() => connectConfigure(jest.fn(), undefined)).toThrow();
+    it('with a render function but no unmount function does not throw', () => {
+      expect(() => connectConfigure(jest.fn(), undefined)).not.toThrow();
     });
 
-    it('with a unmountFn but no renderFn', () => {
-      expect(() => connectConfigure(undefined, jest.fn())).toThrow();
+    it('with a unmount function but no render function does not throw', () => {
```

Comment:
I don't think that case make sense. We should never be in that case.

Reply:
Indeed it doesn't really make sense but it's still a case that should be covered, right?